src/pages/docs/octopus-rest-api/calamari.md

```diff
@@ -1,26 +1,12 @@
 ---
 layout: src/layouts/Default.astro
 pubDate: 2023-01-01
-modDate: 2023-01-01
+modDate: 2024-12-11
 title: Calamari
-description: Calamari is the command-line tool invoked by Tentacle during a deployment. It knows how to extract and install NuGet packages, run the Deploy.ps1 etc. conventions, modify configuration files, and all the other things that happen during an deployment.
 navOrder: 70
+description: Calamari is the command-line tool invoked by Tentacle during a deployment. It knows how to extract and install packages, deploy to Kubernetes, run scripts, conventions, modify configuration files, and all the other things that happen during a deployment.
 ---
 
-Prior to **Octopus 3.0**, Tentacles were responsible for performing deployment steps. Tentacles were *smart*. They knew how to transform configuration files, modify IIS, and much, much more.
```

Comment:
I don't think we need to mention Octopus 3.0 anymore - thoughts?

Reply:
Delete: https://octopusdeploy.slack.com/archives/C04PQ8XJ4CB/p1733827991760049?thread_ts=1733827535.347329&cid=C04PQ8XJ4CB